packages/network/src/index.ts

```diff
@@ -66,6 +66,10 @@ export class Network extends Connector {
 
     const urls = this.urlMap[chainId]
```

Comment:
Could you check @vm how is it possible that TypeScript does not report this as `string[] | undefined`? Like technically speaking, accessing an array by index can return undefined (out of bounds), same for map when key is missing. I don't understand how this passes TypeScript otherwise.